src/main/java/hattrickdata/HattrickDataInfo.java

```diff
@@ -0,0 +1,24 @@
+package hattrickdata;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDateTime;
+
+@Getter
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class HattrickDataInfo {
+
+    private String fileName;
+    private String version;
+    private int userId;
+    private LocalDateTime fetchedDate;
```

Comment:
HODateTime

Reply:
Refactor: Review Finding 5: Use HODateTime instead of LocalDateTime